### DISCUSSION TITLE
The traceid and spanid are not showing completely

### DISCUSSION THREAD

**Discussion by li-zheng-hao** [datalust/seq-tickets#2224] [Help]
I am unable to view the complete trace ID and span ID in the logs on the dashboard page:

![image](https://github.com/datalust/seq-tickets/assets/22129824/cfde9702-f170-4535-89db-618706466a12)

my configuration:

```c#
Log.Logger=new LoggerConfiguration()
    .WriteTo.Console()
    .WriteTo.Seq("http://localhost:5341")
    .CreateLogger();
builder.Host.UseSerilog();

builder.Services.AddOpenTelemetry()
    .ConfigureResource(r => r.AddService(GlobalSystem.ServiceName))
    .WithTracing(tracing =>
    {
        tracing.AddSource(GlobalSystem.SourceName);
        tracing.AddAspNetCoreInstrumentation();
        tracing.AddHttpClientInstrumentation();
        tracing.AddOtlpExporter(opt =>
        {
            opt.Endpoint = new Uri("http://localhost:5341/ingest/otlp/v1/traces");
            opt.Protocol = OtlpExportProtocol.HttpProtobuf;
        });
    });
```

If I use OpenTelemetry Sdk, it works well, but I want to use serilog

> Seq version : 2024.3.11547

**Comment by liammclennan** [comment]
Hi @li-zheng-hao 

Can you please add a little more about what you are trying to do? 

You can get the trace id and span id using the copy actions in the trace menu, or a query like `select @TraceId, @SpanId from stream limit 10 `.

**Reply by li-zheng-hao** [reply]
i just want to view the complete traceid and spanid when i scrolling vertical bar to view logs , but it is trimmed like "abc...". even if I hover the mouse on it , only part of it is displayed , like "@TraceId=abc..." , this is a bit inconvenient, isn't it?

**Reply by li-zheng-hao** [reply]
I think when the mouse hovers over the Traceid, it should show the full content, not the truncated content, it would be a better experience that way

**Reply by liammclennan** [reply]
Thanks. That does sound reasonable, but I'm trying to understand how you are trying to use it. How is the full id helpful? Or how is the shortened version inconvenient?

**Reply by li-zheng-hao** [reply]
For example, I want to find all logs with traceId abc123, and I know they were recorded in the last few minutes. I just need to click on the time filter on the page and set it to 5 minutes. If the page could display the complete traceId, I could quickly find what I need (just by scrolling). But now the traceId only shows abc..., which could be abc456, abc678 or other traceIds. I have to manually enter @TraceId=abc123 in the search box and then click search button, which obviously takes more time.

**Reply by nblumhardt** [reply]
We're reconsidering how we handle trace/span columns; it's not certain yet where it'll land, but thanks for all the feedback either way! :+1:

**Reply by li-zheng-hao** [reply]
Thanks for the work you guys do, seq is a great product and I hope it gets even better!
